infoscreen/templates/infoscreen/base.html

```diff
@@ -3,80 +3,78 @@
 {% load static %}
 {% load compress %}
 
+
 <html lang="en" dir="ltr">
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    
-    
+
+
     <!-- JQuery first (3.5.0 breaks the site), then Popper.js, then Bootstrap JS -->
     <script src="https://code.jquery.com/jquery-3.4.1.js" integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU=" crossorigin="anonymous"></script>
-    {# <script src="https://code.jquery.com/jquery-3.5.0.js" integrity="sha256-r/AaFHrszJtwpe+tHyNi/XCfMxYpbsRg2Uqn0x3s2zc=" crossorigin="anonymous"></script> #}
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js" integrity="sha384-6khuMg9gaYr5AxOqhkVIODVIvm9ynTT5J4V1cfthmT+emCG6yVmEZsRHdxlotUnm" crossorigin="anonymous"></script>
-    {# <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.0/cjs/popper.min.js" integrity="sha256-OH05DFHUWzr725HmuHo3pnuvUUn+TJuj8/Qz9xytFEw=" crossorigin="anonymous"></script> #}
-    {# <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script> #}
-    
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js" integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js" integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13" crossorigin="anonymous"></script>
```

Comment:
Er det et bevisst valg å gå fra `bundle` til bare vanlig?

Reply:
Det er ikkje bevisst, så etter siste versjon i bootstrap sin dokumentasjon. Hva er forskjellen?

Reply:
Mulig bundle inneholder flere tredjepartsbiblioteker, css og js alt på en gang (men er litt usikker, så bør sjekkes). Forsåvidt gyldig å bruke dette også da, men da må vi være bevisst på og fjerne de andre overlappende tingene vi har importert. Vi har typisk pleid å sette styling øverst og scripts nederst. Ligger forklaring i dokumentet håper jeg, hvis ikke kan det kopieres fra rekenett.

Reply:
Du kan gjøre et forsøk med bundle, men da må isåfall popper.js fjernes da bundle har den.

Reply:
https://getbootstrap.com/docs/4.0/getting-started/contents/